src/dynamic-launcher.c

```diff
@@ -680,7 +680,7 @@ handle_request_install_token (XdpDbusDynamicLauncher *object,
    * app was launched from the CLI:
    * https://github.com/flatpak/xdg-desktop-portal/pull/719#issuecomment-1057412221
    */
-  if (xdp_app_info_is_host (call->app_info) && g_str_equal (app_id, ""))
+  if (xdp_app_info_get_kind (call->app_info) == XDP_APP_INFO_KIND_TEST)
```

Comment:
This looks like it'll break the check for unsandboxed apps.

Reply:
unsandboxed apps may have an app-id aren't they? In any case I can check for both HOST and TEST ones to be sure.